components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -305,19 +305,19 @@
     }
   },
   "notificationBackupWalletTitle": {
-    "message": "Backup wallet",
+    "message": "Back up wallet",
```

Comment:
Should this be "Back up your Brave Rewards"?

Reply:
Yep - great catch :)